lib/active_shipping/carriers/fedex.rb

```diff
@@ -136,6 +136,9 @@ class FedEx < Carrier
 
     DEFAULT_LABEL_STOCK_TYPE = 'PAPER_7X4.75'
 
+    # Available return formats for image data when creating labels
+    LABEL_FORMATS = ['DPL', 'EPL2', 'PDF', 'ZPLII', 'PNG'] 
```

Comment:
Are these consumed? Why does this exist? To be able to check `ActiveShipping::Fedex::LABEL_FORMATS` outside of the gem?

If so, it should have a quick test.